examples/platform/ameba/ota/OTAHelper.h

```diff
@@ -0,0 +1,27 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+class OTAHelper
```

Comment:
Please avoid names that are helper/support/util/core in both file names and class names.

What is a `OTAHelper` ? there are no documents and the class name does not help.

T~his seems to initialize an ota requestor. Naming should be as such.

Reply:
OTAHelper has been renamed to OTAInitializer